src/helpers.js

```diff
@@ -63,9 +64,10 @@ export async function sendJob(worker, config) {
 export function showError(givenMessage, givenDetail = null) {
   let detail
   let message
-  if (message instanceof Error) {
```

Comment:
Note that this was actually a :bug:, it was only checking the newly declared `message`, not the one passed in.

Reply:
Good catch